core/jobs/batch_jobs/collection_info_jobs.py

```diff
@@ -0,0 +1,114 @@
+) = models.Registry.import_models([
+    models.NAMES.collection, models.NAMES.feedback, models.NAMES.user])
+
+datastore_services = models.Registry.import_datastore_services()
+
+
+class GetCollectionOwnersEmailsJob(base_jobs.JobBase):
+    """Job that generate emails of CollectionModels users."""
+
+    def run(self) -> beam.PCollection[job_run_result.JobRunResult]:
+        """Returns a PCollection of 'SUCCESS' or 'FAILURE' results from
+        generating email of CollectionModel user.
+
+        Returns:
+            PCollection. A PCollection of 'SUCCESS' or 'FAILURE' results from
+            generating email of CollectionModel user.
+        """
+        collection_user_ids_pcollection = (
+            self.pipeline
+            | 'Get all collection rights models' >> ndb_io.GetModels(
+                collection_models.CollectionRightsModel.get_all(
+                    include_deleted=False))
+            | 'Extract user IDs' >> beam.FlatMap(
+                    lambda collection_rights: collection_rights.owner_ids)
+            | 'Remove duplicates' >> beam.Distinct() # pylint: disable=no-value-for-parameter
+        )
+        collection_user_ids = beam.pvalue.AsIter(
+            collection_user_ids_pcollection)
+
+        user_email_pcollection = (
+            self.pipeline
+            | 'Get all user settings models' >> ndb_io.GetModels(
+                user_models.UserSettingsModel.get_all(
+                    include_deleted=False))
+            | 'Filter model that belong to collection' >> (
+                beam.Filter(
+                    lambda model, ids: (
+                        model.id in ids),
+                    ids=collection_user_ids
+                ))
+            | 'Get e-mail' >> beam.Map(lambda model: model.email)
+        )
+
+        return (
+            user_email_pcollection
+            | 'Count the output' >> (
+                job_result_transforms.CountObjectsToJobRunResult())
+        )
+
+
+class MatchEntiryTypeCollectionJob(base_jobs.JobBase):
+    """Job that match entity_type as collection."""
+
+    def run(self) -> beam.PCollection[job_run_result.JobRunResult]:
+        """Returns a PCollection of 'SUCCESS' or 'FAILURE' results from
+        matching entity_type as collection.
+
+        Returns:
+            PCollection. A PCollection of 'SUCCESS' or 'FAILURE' results from
+            matching entity_type as collection.
+        """
+        feedback_model_matched_as_collection = (
+            self.pipeline
+            | 'Get all GeneralFeedbackThread models' >> ndb_io.GetModels(
+                feedback_models.GeneralFeedbackThreadModel.get_all(
+                    include_deleted=False))
+            | 'Extract entity_type' >> beam.Map(
+                    lambda feeback_model: feeback_model.entity_type)
+            | 'Match entity_type' >> beam.Filter(
+                lambda entity_type: entity_type == 'collection')
+            | 'print out ' >> beam.Map(print)
```

Comment:
Don't use print, rather create the `JobRunResult` model directly and return that.

Reply:
Removed